Controller/Payment/Webhook.php

```diff
@@ -182,22 +182,16 @@ public function execute()
                     $orderWebhookData   = $this->getOrderWebhookData($orderId);
                     $amountPaid         = $post['payload']['payment']['entity']['amount'];
 
+                    $this->setWebhookData($post, $orderWebhookData['entity_id'], true, $paymentId, $amountPaid);
```

Comment:
@nikhilsharma1100 `$this->setWebhookData($post, $orderWebhookData['entity_id'], true, $paymentId, $amountPaid);` function call is included in both `if` / `else` cases, do this call is also required outside before if-else block?

Reply:
@enishant yes, since its an elseif block there are chances none of the blocks execute.